src/gui/editors/PianoRoll.cpp

```diff
@@ -2678,8 +2678,8 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 	f.setPixelSize(m_keyLineHeight * 0.8);
 	p.setFont(f); // font size doesn't change without this for some reason
 	QFontMetrics fontMetrics(p.font());
-	 // G4 is one of the widest
-	QRect const boundingRect = fontMetrics.boundingRect(QString("G4"));
+	// G-1 is one of the widest; plus one pixel for the shadow
+	QRect const boundingRect = fontMetrics.boundingRect(QString("G-1") + 1);
```

Comment:
I think this is wrong. You want to add a pixel to the shadow, not to the string.

Reply:
Yikes, I wonder how did this even not break anything. ^^; Good catch, thanks, I'll fix it tomorrow.